### ISSUE TITLE
I've got "undefined: models in models.User"

### ISSUE THREAD

**Issue by sio4** [gobuffalo/buffalo#383]
Hello,
I am newbee to buffalo and I've got this problem just after I generate my resource by invoking `buffalo g resource User`.

**Environment:**
> $ go version
> go version go1.8.1 linux/amd64
> $ buffalo version
> Buffalo version is: 0.8.1

**Problem:**
I've got following error message and buffalo exit with 2 when I invoke `buffalo dev`:
> actions/users.go:30: undefined: models in models.Users

I solve this issue by adding following:
```patch
--- a/actions/users.go
+++ b/actions/users.go
@@ -3,6 +3,8 @@ package actions
 import (
        "github.com/gobuffalo/buffalo"
        "github.com/markbates/pop"
+
+       "github.com/sio4/myapp/models"
 )
 
 // This file is generated by Buffalo. It offers a basic structure for

```

Is it my fault or bug of generator? I just guess there is some bug on generator.

**Comment by markbates** [comment]
Looks like it's a bug in the generator. The generator is failing to add the `models` package to file.

**Comment by sio4** [comment]
Oh! thanks for lightning fast reply. then am I do right?

**Comment by markbates** [comment]
You're doing it right. If you had `goimports` installed, like a lot of people do, you wouldn't have noticed this bug, as it would've run at the end of the generator and automatically added the import for you. Either way it should add the import manually.

**Comment by sio4** [comment]
Oh! thanks for that information. I am golang newbee too! :-) I will try that later.

**Comment by markbates** [comment]
Fixed in `v0.8.1.2`.

**Comment by sio4** [comment]
Thanks! patch is also lightning fast!
